api/data_explorer/controllers/export_url_controller.py

```diff
@@ -103,23 +107,62 @@ def _get_entities_dict(cohort_name, query):
                 'query': query
             }
         })
+
+    if cohort_name:
+        sample_items = []
```

Comment:
I think it's cleaner if you call _get_doc_generator() here, instead of passing in doc_generator. (And move  _get_doc_generator() to before this method)

Reply:
Oops did this.